test/functional/fundrawtransaction.py

```diff
@@ -453,7 +453,7 @@ def run_test(self):
         self.stop_node(2)
         self.stop_node(3)
         self.nodes[1].encryptwallet("test")
-        self.nodes.pop(1)
+        bitcoind_processes[1].wait(timeout=BITCOIND_PROC_WAIT_TIMEOUT)
```

Comment:
`wait_node(1)` below does this already, should keep one or the other

Reply:
dropped wait_node(1)